internal/tarwriter/tarwriter.go

```diff
@@ -77,7 +77,16 @@ func (tw *tarWriter) Flush() error {
 	}
 	info, _ := os.Stat(fname)
 
-	b, uploadErr := upload.Upload(tw.uploadURL, fname, "application/vnd.redhat.topological-inventory.filename+tgz", tw.metadata)
+	if sha == tw.input.PreviousSHA && info.Size() == tw.input.PreviousSize {
+		err = tw.task.Update(map[string]interface{}{"state": "completed", "status": "unchanged", "message": "Nothing changes since last refresh"})
```

Comment:
@bzwei I think the message should read "Upload skipped, since nothing has changed from last refresh"